gcloud-java-datastore/src/main/java/com/google/gcloud/datastore/testing/LocalGcdHelper.java

```diff
@@ -396,9 +438,8 @@ private void startGcd(Path executablePath) throws IOException, InterruptedExcept
         .command(gcdAbsolutePath.toString(), "start", "--testing", "--allow_remote_shutdown",
             "--port=" + Integer.toString(port), projectId)
         .directory(gcdPath)
-        .redirectErrorStream()
         .start();
-    processReader = ProcessStreamReader.start(startProcess, "Dev App Server is now running");
+    processReader = ProcessStreamReader.start(startProcess, "Dev App Server is now running", true);
```

Comment:
OPT PUT PLEASE

Tiffany Burich

On Oct 14, 27 Heisei, at 10:34 AM, Arie <notifications@github.com> wrote:

In gcloud-java-datastore/src/main/java/com/google/gcloud/datastore/testing/LocalGcdHelper.java:

>          .start();
> -    processReader = ProcessStreamReader.start(startProcess, "Dev App Server is now running");
> +    processReader = ProcessStreamReader.start(startProcess, "Dev App Server is now running", true);
lets create a separate stream readers and pass the stream (out or err) to each one of them.

—
Reply to this email directly or view it on GitHub.